packages/node_modules/@node-red/editor-client/src/js/ui/deploy.js

```diff
@@ -334,6 +334,9 @@ RED.deploy = (function() {
                 var invalidNodes = [];
 
                 RED.nodes.eachConfig(function(node) {
+                    if (!node.hasOwnProperty("valid")) {
```

Comment:
`node.hasOwnProperty` should be replaced with `Object.prototype.hasOwnProperty.call(node, "valid")` 

This is the recommended form of testing for a property that avoids errors if the `object` has no `[[Prototype]]` chain

Reply:
Thank you for your suggestion.
I fixed it not to use `hasOwnProperty`.  However, I feel that `Object.prototype.hasOwnProperty.call (node," valid ")` is too long 😵‍💫 so I adopted `undefined` check.
It would be nice to have a utility such as `RED.util.hasOwnProperty`.

Reply:
> node.hasOwnProperty should be replaced with Object.prototype.hasOwnProperty.call(node, "valid")

It will depend on where the `object` has come from. If it is one we are creating, then we know it has a prototype because we don't create null objects. In this instance, we know `node` is vanilla JavaScript Object that has been generated by `JSON.parse()`.